mkosi/__init__.py

```diff
@@ -40,32 +40,34 @@
 from textwrap import dedent
 from types import FrameType
 from typing import (
-    IO,
-    TYPE_CHECKING,
```

Comment:
That's what isort is complaining about.

Reply:
Oh, I see. It thinks that `IO` needs to go first because it's all-capitals, but `BinaryIO` can go in the usual place, because it's clearly completely different than `IO`. Oh, well. In effect, this order seems semi-random. I don't like automated tools that override logic.

Anyway, I dropped the commit and applied isort to keep the order that it likes in individual commits.

Reply:
I think the idea is: 
1. `CONSTANTS`
2. `Classes`
3. `functions`
and alphabetically within.